packages/api/src/transaction/dtos/transaction.dto.ts

```diff
@@ -14,7 +14,7 @@ export class TransactionDto {
     description: "The address this transaction is to",
     example: "0xc7e0220d02d549c4846A6EC31D89C3B670Ebe35C",
   })
-  public readonly to: string;
+  public readonly to?: string;
```

Comment:
Why is it optional?

Reply:
Initially we where supporting to as a null when we started with evm support. I will change that back